backend/corpus_asset_pipelines/integrated_corpus/transform.py

```diff
@@ -37,6 +38,32 @@ def apply_pre_concatenation_filters(
     return anndata_object
 
 
+def create_high_level_tissue(anndata_object: anndata.AnnData):
+    anndata_object.obs["tissue_original"] = anndata_object.obs["tissue"]
+    anndata_object.obs["tissue_original_ontology_term_id"] = anndata_object.obs["tissue_ontology_term_id"]
+    anndata_object.obs = get_high_level_tissue(anndata_object.obs)
+
+
+# TODO finalize this function
+def get_high_level_tissue(obs: DataFrame):
```

Comment:
I think we should finalize this function in this PR, as I would prefer to avoid hardcoding the tissue rollup (even if it is temporary).

Reply:
I was planing to keep them separate for better and encapsulated review. 

That work is done in the branch pablo-gar/wmg-tissue-roll-up-tissueMapper

[](https://github.com/chanzuckerberg/single-cell-data-portal/blob/pablo-gar/wmg-tissue-roll-up-tissueMapper/backend/wmg/data/tissue_mapper.py)

[](https://github.com/chanzuckerberg/single-cell-data-portal/blob/d708135988ed5da2614a66a881d41550ce506583/backend/corpus_asset_pipelines/integrated_corpus/transform.py#L45-L74)